src/osx/webview_webkit.mm

```diff
@@ -623,43 +524,39 @@ @interface WebViewCustomProtocol : NSURLProtocol
 
 wxString wxWebViewWebKit::GetSelectedSource() const
 {
-    DOMRange* dr = [m_webView selectedDOMRange];
-    if ( !dr )
-        return wxString();
-
-    return wxCFStringRef::AsString([dr markupString]);
+    // TODO: not implemented in SDK (could probably be implemented by script)
```

Comment:
This is a rather important operation (UI often provides operations working with the selected text), so if implementing it is as simple as doing `RunScript("window.getSelection().innerText")` or something like this, it would be nice to do it.

Reply:
I don't think `GetSelectedSource` is a very common operation. Surly `GetSelectedText` is, which is implemented. Your suggestion doesn't return the source but just the text, which `GetSelectedText` already provides.

Reply:
Oops, sorry, I've indeed just mixed up the two functions, sorry.